test/data/index.js

```diff
@@ -45,7 +45,11 @@ const testData = Object.assign( // eslint-disable-line prefer-object-spread
   Users
 );
 
-testData.seed = seed;
-testData.reset = resetDataStores;
+seed();
+
+testData.reset = () => {
+  resetDataStores();
+  seed();
+};
```

Comment:
If there's ever a need in the future to be able to reset without reseeding, we can add an optional parameter to `testData.reset()` along those lines. However, it's not something we've needed so far.